src/NumericalAlgorithms/DiscontinuousGalerkin/Protocols.hpp

```diff
@@ -112,16 +113,17 @@ struct IsNumericalFluxCallable
  * normal.
  *
  * Here's an example for a simple "central" numerical flux
- * \f$G(\boldsymbol{n}^\mathrm{int}, u^\mathrm{int},
- * \boldsymbol{n}^\mathrm{ext}, u^\mathrm{ext}) = \frac{1}{2}\left(
- * u_\mathrm{int} - n^\mathrm{int}_i n_\mathrm{ext}^i u_\mathrm{ext} \right)\f$
- * for the case where the interface normal is independent of the system
- * variables so
- * \f$\boldsymbol{n}_\mathrm{ext} = -\boldsymbol{n}_\mathrm{int}\f$
- * and therefore
- * \f$G=\frac{1}{2}\left(u_\mathrm{int} + u_\mathrm{ext} \right)\f$:
+ * \f$G(n_i^\mathrm{int}, u^\mathrm{int}, n_i^\mathrm{ext}, u^\mathrm{ext}) =
+ * \frac{1}{2}\left( n_i^\mathrm{int} F^i(u_\mathrm{int}) - n_i^\mathrm{ext}
+ * F^i(u_\mathrm{ext}) \right)\f$:
```

Comment:
You still use `u_{\mathrm{int}}`. It should be `u^{\mathrm{int}}` (same for ext)